### ISSUE TITLE
DB/Quest: improve cosmetic details of quest In Service Of The Lich King

### ISSUE THREAD

**Issue by Wyreth** [TrinityCore/TrinityCore#20809]
**Description:**

The quest [In Service Of The Lich King](http://www.wowhead.com/quest=12593) is missing correct emote delays in quest_details. Also, the Lich King voiceover that happens on quest accept should be handled by a spell.

**Branch(es):** both.

**TC rev. hash/commit:** 28cd4b5

**TDB version:** 335.63

**Suggested fix**:
```sql
UPDATE `quest_details` SET `EmoteDelay1`=500, `EmoteDelay2`=1000 WHERE `ID`=12593;
UPDATE `smart_scripts` SET `action_type`=11, `action_param1`=58204, `target_type`=7, `comment`="The Lich King - On Quest 'In service of The Lich King' Taken - Cast 'LK Intro VO'" WHERE `entryorguid`=25462 AND `source_type`=0;
DELETE FROM `smart_scripts` WHERE `entryorguid`=2546200 AND `source_type`=9;


```

**Comment by offl** [comment]
58024 shouldn't be used?

**Comment by Wyreth** [comment]
I'm not quite sure (had this fix lying around for a while, don't have the sniff at hand), but that spell has a visual I've never seen before. It's probably a leftover from Blizzard :P
